base/shell/explorer/taskswnd.cpp

```diff
@@ -489,34 +489,28 @@ class CTaskSwitchWnd :
 
     HICON GetWndIcon(HWND hwnd)
     {
-        HICON hIcon = NULL;
+        HICON hIcon;
 #define GET_ICON(type) \
-    SendMessageTimeout(hwnd, WM_GETICON, (type), 0, SMTO_ABORTIFHUNG, 100, (PDWORD_PTR)&hIcon)
+    (HICON)SendMessageW(hwnd, WM_GETICON, (type), 0)
```

Comment:
No, we should not hang the shell in case a window hangs!

Reply:
Fixed.